pymc3/distributions/dist_math.py

```diff
@@ -365,34 +365,31 @@ def conjugate_solve_triangular(outer, inner):
             grad = tt.triu(s + s.T) - tt.diag(tt.diagonal(s))
         return [tt.switch(ok, grad, floatX(np.nan))]
 
+
 class SplineWrapper (theano.Op):
     """
     Creates a theano operation from scipy.interpolate.UnivariateSpline
     """
 
-    __props__ = ('spline',)
+    __props__ = ('spline', 'grad_op')
     itypes = [tt.dscalar]
     otypes = [tt.dscalar]
 
-    def __init__(self, spline):
+    def __init__(self, spline, n_derivatives):
```

Comment:
`n_derivatives` is the number of derivatives that a given `SplineWrapper` object should implement. One wouldn't want to have it larger than necessary,  because invocations of `self.spline.derivative()` are slow and have O(n) complexity, infinite recursion doesn't seem to be a good choice here.